plugins/pdr_deterministic_plugin/ufm_sim_web_service/api/pdr_plugin_api.py

```diff
@@ -110,20 +111,19 @@ def include_ports(self):
 
         return response, HTTPStatus.OK
 
-
-    def get_request_data(self):
+    @staticmethod
+    def get_request_data():
         """
         Deserialize request json data into object
         """
         if request.is_json:
             # Directly convert JSON data into Python object
             return request.get_json()
-        else:
-            # Attempt to load plain data text as JSON
-            return json.loads(request.get_data(as_text=True))
-
+        # Attempt to load plain data text as JSON
+        return json.loads(request.get_data(as_text=True))
 
-    def fix_port_name(self, port_name):
+    @staticmethod
+    def fix_port_name(port_name):
```

Comment:
Why did you convert it to a static class?

Reply:
reverted